test/v2-migration-tests/src/test/resources/software/amazon/awssdk/v2migrationtests/maven/after/src/main/java/foo/bar/S3RequestConstructor.java

```diff
@@ -199,12 +199,12 @@ public void requestconstructor() {
             .build());
 
         //INCOMPATIBLE RESPONSE
```

Comment:
- Do we need both comments `//INCOMPATIBLE RESPONSE` and the new one ?
- Also if the newly added comment is single line can we use` // `  Single-line comment  for the new comment added